mods/doors/init.lua

```diff
@@ -437,18 +437,26 @@ function doors.register(name, def)
 	def.selection_box = {type = "fixed", fixed = {-1/2,-1/2,-1/2,1/2,3/2,-6/16}}
 	def.collision_box = {type = "fixed", fixed = {-1/2,-1/2,-1/2,1/2,3/2,-6/16}}
 
-	def.mesh = "door_a.obj"
-	minetest.register_node(":" .. name .. "_a", def)
+	if def.model then
+		def.mesh = def.model .. "_a.obj"
+		minetest.register_node(":" .. name .. "_a", def)
 
-	def.mesh = "door_b.obj"
-	minetest.register_node(":" .. name .. "_b", def)
+		def.mesh = def.model .. "_b.obj"
+		minetest.register_node(":" .. name .. "_b", def)
+	else
+		def.mesh = "door_a.obj"
+		minetest.register_node(":" .. name .. "_a", def)
+
+		def.mesh = "door_b.obj"
+		minetest.register_node(":" .. name .. "_b", def)
+	end
```

Comment:
Suggestion:
```lua
def.model = def.model or "door"

def.mesh = def.model .. "_a.obj"
...
```

Reply:
Why need `def.model = def.model or "door"`? The mod simply specifies nothing for the default model to be used.
Done the latter already.

Reply:
If `def.model` is `nil` (which means that it is not specified), then `def.model or "door"` is `"door"`.
You do not need so many `if`s and duplicated code this way.

Reply:
I think DS-Minetest may be correct.

Reply:
OK, I'll do that.

Reply:
`if def.model == def.model or "door" then`
Doesn't seem right and isn't what was suggested.

Reply:
Then what goes in the condition? I'm confused. Could you explain further, please?

Reply:
Will do!